openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js

```diff
@@ -1258,27 +1257,21 @@ function thematicMapCreation() {
     $('#absoluteSpinner').hide();
 }
 
-function watchForPdSelection() {
+function whenProjDefSelected() {
+    $('#pdSelection').prop("disabled", true);
```

Comment:
The '#pdSelection' probably must be disabled inside the fuction fired by '#submitPd'  click and reenabled after .post async call returned.